src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -1665,6 +1825,45 @@ components:
+      format: float
+      minimum: -90.0
+      maximum: 90.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
+      example: 24
+
+    rotationY:
+      type: number
+      format: float
+      minimum: -180.0
+      maximum: 180.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
+      example: 24
+
+    rotationZ:
+      type: number
+      format: float
+      minimum: -180.0
+      maximum: 180.0
+      description: The rotation reading contains 3 angles, measured in degrees that define the orientation of the device in three-dimensional space. These angles are similar to yaw, pitch and roll but are defined using only right hand rotation with axes as defined by the right hand cartesian coordinate system. The three angles are applied to the device in the following order. Right-handed rotation z (-180, 180]. Starting from the y-axis and incrementing in the counter-clockwise direction. Right-handed rotation x [-90, 90]. Starting from the new (once-rotated) y-axis and incrementing towards the z-axis. Right-handed rotation y (-180, 180]. Starting from the new (twice-rotated) z-axis and incrementing towards the x-axis.
+      example: 24
+
+    fieldOfView:
+      type: number
+      format: float
+      minimum: 0
+      maximum: 360
+      description: The field of view of the sensor in degrees.  If absent, then this assumes that the sensor has a 360 degree field of view (i.e. omnidirectional antenna).
+      example: 40
+
+    clipFar:
+      type: integer
+      format: float
+      minimum: 0
+      description: The furthest distance in meters that the sensor is capable of detecting a transmitter.
```

Comment:
Would this field be better indicated by "received sensitivity" in units of dBm? Distance in meters is dependent on a lot of factors so it is hard to specific a value here. However, if we use receiver sensitivity then radio link budget can be used to calculate the distance in meters with the environmental variables. I have a good equation I have used in other code that I can dig up if you are interested.

Reply:
name is changed to receivedSentitivity.  Should the equation/algorithm be embedded in the comments for receivedSensitivity?  If so, let me know and I'll include it.